.gitattributes

```diff
@@ -31,3 +31,6 @@ assets/img/* export-ignore
 *.scala text eol=lf
 *.xml text eol=lf
 *.py text eol=lf
+common/src/test/resources/ssl/generate_certs.sh text
+common/src/test/resources/ssl/* -text
+worker/src/test/resources/ssl/* -text
```

Comment:
how about?
```suggestion
*.jks text
```

Reply:
seems it's not necessary to add a hint for sh, or will it cause issues?

Reply:
The idea is the reverse - to mark jks as binary file, not text.